docs/api-reference/full-text/bm25.mdx

```diff
@@ -250,3 +250,43 @@ FROM <index_name>.search(
 <ParamField body="order_by_direction" default={"asc"}>
   The sort order to use.
 </ParamField>
+
+## Lenient Parsing
+
+Lenient parsing is disabled by default. To enable it, set the `lenient_parsing` parameter to `true`. This setting parses invalid query on a best effort basis.
+
+```sql
+SELECT * FROM <index_name>.search(
+  '<query>',
+  lenient_parsing => true
+);
+```
+
+<ParamField body="index_name" required>
+  The name of the index.
+</ParamField>
+<ParamField body="query" required>
+  The query string.
+</ParamField>
+<ParamField body="lenient_parsing" default={false}>
+  A boolean specifying whether ParadeDB should perform lenient parsing of the
+  query string.
+</ParamField>
+
+## Conjunction By Default
+
+By default, the `conjunction_by_default` parameter is set to false. To change this behavior and enable conjunction mode, you need to set the `conjunction_by_default` parameter to `true`. When enabled, the query composition will use conjunction (AND) by default instead of disjunction (OR).
```

Comment:
Should put `false` in backticks: "\`false\`" for syntax highlighting.

Reply:
I think specific examples of query + results would be helpful here. It's a little hard for a user to picture how this works without seeing an actual queyr.

I'm also not sure about the name of the option. I realize this is Tantivy's name, but we might consider deviating here. It's not clear what "by_default" means, it implies that this can be later overridden but I'm not sure that's true.

Reply:
I also noticed that. I renamed the parameter to `conjunction_mode`, and added an example usage now.